### PR TITLE
[Feature] Allows to pretty print ClassDefinitions

### PR DESCRIPTION
**Pull request by benammann**
Sometimes it happens that you have to merge the class definitions. But the native format of `var_export` causes some problems here.

The library https://github.com/brick/varexporter makes the created PHP code much more readable and mergeable.

This feature can be activated with the environment variable `PIMCORE_CLASS_DEFINITION_PRETTY_PRINT=true`.

Once enabled the class definitions are created via the `brick/varexporter` library.

### CODE REVIEW

**Comment by brusch** [comment]
Nice, thanks a lot for your PR! 
Just 2 more things: 
- I don't think we need an option to turn that on (if it is necessary for whatever reason, it should be a Symfony Config instead - not a constant) 
- If we use `VarExporter` instead of `var_export` we should take the chance to check the export/loading in a unit test, just to be sure it works for all data-types

Thanks!

File: composer.json [hunk 1]
```diff
@@ -95,7 +95,8 @@
     "scheb/two-factor-bundle": "^3.26",
     "phpoffice/phpspreadsheet": "^1.9",
     "html2text/html2text": "^4.2.1",
-    "webmozarts/console-parallelization": "^1.1.0"
+    "webmozarts/console-parallelization": "^1.1.0",
+    "brick/varexporter": "^0.3.2"
```
Comment (jdreesen) [inline composer.json:99]: This should be an optional dependency, I think, since it's only used in one place and on demand.

**Comment by brusch** [comment]
Just a note for later reference: If the 2 points mentioned above are getting fixed and this PR get's merged, we should replace all usages of `var_export_pretty()` with `VarExporter` in a follow up PR.

**Comment by benammann** [comment]
Hi @brusch @jdreesen,

thanks for your feedback!

As soon as I have time again I will take care of the points and update the pull request.

**Comment by brusch** [comment]
@benammann any updates on this?

**Comment by brusch** [comment]
Closed due to inactivity. 
Please feel free to come back on this PR as soon as you have time again 😊
Thanks and happy new year! 🎉

**Pull request closed without merging**